macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -89,6 +91,18 @@ open class AvatarView: NSView {
 		}
 	}
 
+	/// The initials font color of the avatar view when no image is provided.
+	/// Setting this property will override the default provided color.
+	@objc open var initialsFontColor: NSColor {
+		didSet {
+			guard oldValue != initialsFontColor else {
+				return
+			}
+			needsDisplay = true
+			initialsTextField.textColor = initialsFontColor
+		}
```

Comment:
Do we need to update the updateLayer() method in line 76?

That method is also called every time a light/dark switch happens btw, and is the Apple-accepted way of setting dynamic colors to a layer based view

Reply:
Thanks Saad for bringing this to my attention, it doesn't seem this is related to `foregroundColor`. This is base view for avatar. I have highlighted the base view in the image below. 

![Screen Shot 2021-03-04 at 12 36 24 PM](https://user-images.githubusercontent.com/63682282/110027239-93449600-7ce6-11eb-902e-47d41391ab8b.png)

It does get called during theme switch so does any other view layout methods. Based on my understanding it doesn't need to be updated.